service/ocean/providers/aws/cluster.go

```diff
@@ -175,6 +175,7 @@ type LaunchSpecification struct {
 	EBSOptimized             *bool                    `json:"ebsOptimized,omitempty"`
 	UseAsTemplateOnly        *bool                    `json:"useAsTemplateOnly,omitempty"`
 	InstanceMetadataOptions  *InstanceMetadataOptions `json:"instanceMetadataOptions,omitempty"`
+	LaunchSpecScheduling     *LaunchSpecScheduling    `json:"scheduling,omitempty"`
```

Comment:
@yehielnetapp 
The object name should be 'scheduling' and not 'LaunchSpecScheduling'. The object type can stay the same.
What about the other parameters?
Don't we need to pass the 'Tasks' entities inside the 'scheduling'?